src/components/Property/Property.jsx

```diff
@@ -4,44 +4,67 @@ import PropertyDetail from './PropertyDetail';
 import PropertyCalender from './PropertyCalender';
 import PropertyReview from './PropertyReview';
 import PropertyMap from './PropertyMap';
+import PropertyReservation from './PropertyReservation';
+import PropertyOthers from './PropertyOthers';
+import PropertyHost from './PropertyHost';
 import styled from 'styled-components';
 import {
   flexSpaceBetweenCenter,
   flexColumnCenter,
   theme,
 } from '../../styles/theme';
 import axios from 'axios';
-
-import { FaMedal } from 'react-icons/fa';
+import { DETAIL_API } from '../../config'
 import { MdStar } from 'react-icons/md';
-import { HiShieldCheck } from 'react-icons/hi';
+import { FaMedal } from 'react-icons/fa';
 import { FiShare2 } from 'react-icons/fi';
 import { BsHeart } from 'react-icons/bs';
 
 const Property = (props) => {
   const [propertyImages, setPropertyImages] = useState([]);
+  const [propertyInfo, setPropertyInfo] = useState({});
 
   useEffect(() => {
     axios
       .get('/data/propertyImages.json')
       .then(({ data: { image } }) => setPropertyImages(image));
   }, []);
 
+  useEffect(() => {
+    axios
+      .get('/data/property.json')
+      // .get(DETAIL_API)
+      .then(({ data: { result } }) => setPropertyInfo(result));
+  }, []);
```

Comment:
axios 이용하셨네요! 굿굿! 👏🏻 
useEffect 를 하나로 사용하셔도 좋을 것 같아요! 

```suggestion
  useEffect(() => {
    axios
      .get('/data/propertyImages.json')
      .then(({ data: { image } }) => setPropertyImages(image));
    axios
      .get('/data/property.json')
      // .get(DETAIL_API)
      .then(({ data: { result } }) => setPropertyInfo(result));
  }, []);
```

Reply:
우와 몰랐어요...!수정했습니다 감사합니다 신영님 :)